src/runtime/index.js

```diff
@@ -49,11 +49,15 @@ export default class Heimdall {
     this._session.events.push(OP_ANNOTATE, NULL_NUMBER, NULL_NUMBER, info);
   }
 
+  hasMonitor(name) {
+    return Boolean(this._monitors.has(name));
```

Comment:
what's the reason for `Boolean` here?  seems fine to just return `this._monitors.has(name)` and rely on truthiness, no?